--- rrdtool-1.2.13/configure.ac.orig	2006-05-04 15:26:29.000000000 +0200
+++ rrdtool-1.2.13/configure.ac	2006-05-28 10:10:20.253466500 +0200
@@ -632,7 +632,7 @@
   withval=""
   AC_ARG_WITH(tcllib,[  --with-tcllib=DIR       location of the tclConfig.sh])
   enable_tcl=no
-  for dir in $withval /usr/lib /usr/local/lib /usr/lib/tcl8.4 /usr/lib/tcl8.3 ; do
+  for dir in $withval /usr/lib /usr/local/lib /usr/lib/tcl8.5 /usr/lib/tcl8.4 /usr/lib/tcl8.3 ; do
     AC_MSG_CHECKING(for tclConfig.sh in $dir)
     if test -f "$dir/tclConfig.sh" ; then
     	tcl_config=$dir/tclConfig.sh
@@ -654,7 +654,7 @@
 	fi
   fi
   AC_ARG_ENABLE(tcl,[  --enable-tcl-site        install the tcl extension in the tcl tree],
-  [],[enable_tcl_site=yes])
+  [enable_tcl_site="$enable_tcl"],[enable_tcl_site=yes])
 
 fi
 

