diff -ruN rrdtool-1.0.28.orig/doc/Makefile.am rrdtool-1.0.28/doc/Makefile.am
--- rrdtool-1.0.28.orig/doc/Makefile.am	Sat Apr 29 15:18:28 2000
+++ rrdtool-1.0.28/doc/Makefile.am	Sun Oct  8 21:20:43 2000
@@ -27,7 +27,7 @@
 idoc_DATA = $(POD) $(TXT)
 ihtmldir = $(prefix)/html
 ihtml_DATA = $(HTML)
-imandir = $(prefix)/man/man1
+imandir = $(mandir)/man1
 iman_DATA = $(MAN)
 
 all-local: link txt man html
