--- rrdtool-2003-12-11/bindings/Makefile.am.orig	Mon Jul 28 23:59:22 2003
+++ rrdtool-2003-12-11/bindings/Makefile.am	Thu Dec 11 11:13:23 2003
@@ -6,11 +6,11 @@
 # lets schedule the perl stuff for installation
 # the special call to install-sh is because the -d switch is not portable
 install-data-local:
-	../config/mkinstalldirs $(DESTDIR)$(prefix)/lib/perl/auto/RRDs
-	$(INSTALL) -m 644 perl-piped/RRDp.pm $(DESTDIR)$(prefix)/lib/perl
-	$(INSTALL) -m 644 perl-shared/RRDs.pm $(DESTDIR)$(prefix)/lib/perl
-	$(INSTALL) -m 644 perl-shared/blib/arch/auto/RRDs/RRDs.bs $(DESTDIR)$(prefix)/lib/perl/auto/RRDs
-	$(INSTALL) -m 755 perl-shared/blib/arch/auto/RRDs/RRDs.@SO_EXT@ $(DESTDIR)$(prefix)/lib/perl/auto/RRDs
+	../config/mkinstalldirs $(DESTDIR)$(perl_sitearch)/auto/RRDs
+	$(INSTALL) -m 644 perl-piped/RRDp.pm $(DESTDIR)$(perl_sitearch)
+	$(INSTALL) -m 644 perl-shared/RRDs.pm $(DESTDIR)$(perl_sitearch)
+	$(INSTALL) -m 644 perl-shared/blib/arch/auto/RRDs/RRDs.bs $(DESTDIR)$(perl_sitearch)/auto/RRDs
+	$(INSTALL) -m 755 perl-shared/blib/arch/auto/RRDs/RRDs.@SO_EXT@ $(DESTDIR)$(perl_sitearch)/auto/RRDs
 
 # add the following to the all target
 all-local:      @COMP_PERL@
