--- rrdtool-1.2.0/configure.ac.orig	2005-04-25 22:56:40.000000000 +0200
+++ rrdtool-1.2.0/configure.ac	2005-04-26 23:03:21.551265304 +0200
@@ -222,7 +222,7 @@
 AM_CONDITIONAL(BUILD_RRDCGI,[test $enable_rrdcgi != no])
 
 if test $enable_rrdcgi != no; then
-EX_CHECK_ALL(cgi,        cgiInit,                   cgi.h,                  cgilib,      0.5,    http://www.infodrom.org/projects/cgilib, "")
+EX_CHECK_ALL(cgic,        cgiInit,                  cgi.h,                  cgilib,      0.5,    http://www.infodrom.org/projects/cgilib, "")
 fi
 EX_CHECK_ALL(art_lgpl_2, art_vpath_add_point,       libart_lgpl/libart.h,   libart-2.0,  2.3.17, ftp://ftp.gnome.org/pub/GNOME/sources/libart_lgpl/2.3/, /usr/include/libart-2.0)
 EX_CHECK_ALL(z,          zlibVersion,               zlib.h,                 zlib,        1.2.1,  http://www.gzip.org/zlib/, "")
