diff -Nur rrdtool-1.0.42.orig/gd1.3/Makefile.am rrdtool-1.0.42/gd1.3/Makefile.am
--- rrdtool-1.0.42.orig/gd1.3/Makefile.am	2003-04-04 23:23:55.000000000 +0200
+++ rrdtool-1.0.42/gd1.3/Makefile.am	2003-04-17 20:48:34.000000000 +0200
@@ -3,15 +3,14 @@
 noinst_LTLIBRARIES  = librrd_gd.la
 
 librrd_gd_la_SOURCES = \
-	gd.c		gdfontg.c	gdfontl.c	gdfontmb.c	\
-	gdfonts.c	gdfontt.c	gdlucidab10.c	gdlucidab12.c	\
+	gdlucidab10.c	gdlucidab12.c	\
 	gdlucidab14.c	gdlucidan10.c	gdlucidan12.c	gdlucidan14.c \
 	gdlucidan10l2.c  gdlucidab12l2.c \
-	gd.h           gdfontmb.h     gdlucidab10.h  gdlucidan10.h \
-	gdfontg.h      gdfonts.h      gdlucidab12.h  gdlucidan12.h \
-	gdfontl.h      gdfontt.h      gdlucidab14.h  gdlucidan14.h \
+	gdlucidab10.h  gdlucidan10.h \
+	gdlucidab12.h  gdlucidan12.h \
+	gdlucidab14.h  gdlucidan14.h \
 	gdlucidan10l2.h  gdlucidab12l2.h
-
+librrd_gd_la_LIBADD = -lgd
 
 EXTRA_DIST= README.rrdtool demoin.gif gd.dsp gd.dsw index.html readme.txt \
 	webgif.c mtables.c mathmake.c giftogd.c
diff -Nur rrdtool-1.0.42.orig/Makefile.am rrdtool-1.0.42/Makefile.am
--- rrdtool-1.0.42.orig/Makefile.am	2003-04-17 20:46:54.000000000 +0200
+++ rrdtool-1.0.42/Makefile.am	2003-04-17 20:47:27.000000000 +0200
@@ -16,7 +16,7 @@
 endif
 
 # build the following subdirectories
-SUBDIRS = cgilib-0.4 gd1.3 $(SUBDIRS_1) $(SUBDIRS_2) \
+SUBDIRS = cgilib-0.4 gd1.3 \
           src doc examples contrib tcl
 
 # the following files are not mentioned in any other Makefile
diff -Nur rrdtool-1.0.42.orig/src/Makefile.am rrdtool-1.0.42/src/Makefile.am
--- rrdtool-1.0.42.orig/src/Makefile.am	2002-05-07 23:51:19.000000000 +0200
+++ rrdtool-1.0.42/src/Makefile.am	2003-04-17 20:49:49.000000000 +0200
@@ -34,7 +34,7 @@
 
 
 
-AM_CPPFLAGS = -I$(CGI_INC_DIR) -I$(GD_INC_DIR) $(PNG_INC) $(ZLIB_INC)
+AM_CPPFLAGS = -I$(CGI_INC_DIR) -I$(GD_INC_DIR)
 
 #COMPILE   = $(CC) $(DEFS) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) $(CFLAGS_EXTRA)
 #LTCOMPILE = $(LIBTOOL) --mode=compile $(CC) $(DEFS) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS) $(CFLAGS_EXTRA)
@@ -75,8 +75,7 @@
 RRD_LIBS =				\
 	$(CGI_LIB_DIR)/librrd_cgi.la	\
 	$(GD_LIB_DIR)/librrd_gd.la	\
-	$(PNG_LIB)			\
-	$(ZLIB_LIB)
+	-lpng -lz
 
 lib_LTLIBRARIES           = librrd.la
 noinst_LTLIBRARIES        = librrd_private.la
--- rrdtool-1.0.42/perl-shared/Makefile.PL.in.orig	2002-03-21 22:52:35.000000000 +0100
+++ rrdtool-1.0.42/perl-shared/Makefile.PL.in	2003-04-17 21:07:29.000000000 +0200
@@ -4,7 +4,7 @@
 # the contents of the Makefile that is written.
 
 # Specify the location of the archive containing PIC compiled object files.
-my $librrd = "-L../src/.libs/ @RRD_PRIVATE_LINK@"  ;
+my $librrd = "-L../src/.libs/ -lrrd"  ;
 
 WriteMakefile(
     'NAME'         => 'RRDs',
@@ -13,7 +13,6 @@
     'CCFLAGS'      => '-DHAVE_CONFIG_H',
     # where to look for the necessary libraries 
     # Perl will figure out which one is valid
-    'depend'	   => {'RRDs.c' => "@top_srcdir@/src/.libs/librrd_private.a"},
     'dynamic_lib'  => {'OTHERLDFLAGS' => "$librrd -lm"},
     'realclean'    => {FILES => 't/demo?.rrd t/demo?.gif' }
 );
--- rrdtool-1.0.44/tcl/Makefile.am.orig	2002-02-26 11:21:52.000000000 +0100
+++ rrdtool-1.0.44/tcl/Makefile.am	2003-04-17 21:07:51.000000000 +0200
@@ -22,7 +22,7 @@
 if COMP_TCL
 
 $(TCL_LIB_FILE): tclrrd.o
-	$(TCL_SHLIB_LD) $(LIBDIRS) $< -o $@ @RRD_PRIVATE_LINK@ -lm $(TCL_LIB_SPEC)
+	$(TCL_SHLIB_LD) $(LIBDIRS) $< -o $@ -lrrd -lm -ltcl
 
 tclrrd.o: tclrrd.c
 	$(CC) $(CFLAGS) $(TCL_SHLIB_CFLAGS) $(INCLUDES) -c $< -DVERSION=\"$(VERSION)\"
